PAWS/PAWSStandard.swift

```diff
@@ -231,6 +231,15 @@ actor PAWSStandard: Standard, EnvironmentAccessible, HealthKitConstraint, Onboar
         guard let accountStorage else {
             preconditionFailure("Account Storage was requested although not enabled in current configuration.")
         }
+        if let dob = details.dateOfBrith {
+            let ageComponents = Calendar.current.dateComponents([.year], from: dob, to: .now)
+            // Store whether the participant is older or younger than 18.
+            if let age = ageComponents.year,
+               let ageGroupData = try? JSONEncoder().encode(age >= 18) {
+                _ = try await userBucketReference.child("ageGroup/isAdult").putDataAsync(ageGroupData)
```

Comment:
I think you are putting the data in the Firebase cloud storage bucket and not Firebase Firestore live database. I would suggest to put the value in the user document in the Firestore.